src/DetailsView/reports/components/report-sections/details-section.scss

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+@import '../../../../common/styles/colors.scss';
+
+@mixin boxShadow {
```

Comment:
does it make sense to have this `mixin` here now? I only see one usage of it

Reply:
It was giving me compilation error and I did not want to import the `automated-checks-report.scss` here (to avoid cyclic dependency). Open to feedback on how to make it better!